### DISCUSSION TITLE
LFS still using pointer

### DISCUSSION THREAD

**Discussion by Fraccas** [raxod502/heroku-buildpack-git-lfs#11] [Q&A] [answered]
Hi team, 

I think I've successfully set up this buildpack on Heroku, however, I'm still getting the LFS file as a pointer instead of the actual binary which is causing the Unity WebGL initialization to fail. 

![image](https://user-images.githubusercontent.com/3107701/175789472-52576a7f-1cca-40e8-aa13-a3bd35ede38c.png)

Maybe this solution does not solve the problem I'm experiencing. The WebGL.data file exceeds 100MB so I'm using LFS to push the file to git/Heroku. I need the binary to actually exist so that the game will load the information from the file. 

![image](https://user-images.githubusercontent.com/3107701/175789504-2e469ecc-3fc6-4d38-8cb7-471a95f4df50.png)

Config Var:
BUILDPACK_GIT_LFS_REPO: 
https://access_token@github.com/Fraccas/FracctalMarketplace.git

EDIT:
I was actually able to find the hosted file directly through GitHub and point the site directly to it, though this method seems to be extremely slow so we'll likely be looking for an alternative solution. 
![image](https://user-images.githubusercontent.com/3107701/175834437-785e4782-c632-4f4c-abda-db3482aa6c50.png)

**Comment by Fraccas** [comment, marked as answer]
We've moved our large files to AWS which seems to be a good solution for us.

**Reply by raxod502** [reply]
Yeah, to be honest I think that is a better solution. Large deploy slugs are not a good architectural pattern and that is not something I fully understood when I created this project back in 2019.

**Comment by raxod502** [comment]
Unfortunately I'm not sure why this might have been happening. Sorry!
